src/pkg/cli/composeDown.go

```diff
@@ -53,7 +53,8 @@ func ComposeDown(ctx context.Context, projectName string, c client.FabricClient,
 
 	delegateDomain, err := c.GetDelegateSubdomainZone(ctx)
 	if err != nil {
-		term.Debug("Failed to get delegate domain:", err)
+		term.Debug("GetDelegateSubdomainZone failed:", err)
+		return "", errors.New("failed to get delegate domain")
```

Comment:
I thought we had a linter set up to catch this, but maybe we only enabled it in the mvp repo. I'll look into that